packages/chains/package.json

```diff
@@ -18,7 +14,7 @@
   "scripts": {
     "?pack-release": "When https://github.com/changesets/changesets/issues/432 has a solution we can remove this trick",
     "pack-release": "rimraf ./_release && yarn pack && mkdir ./_release && tar zxvf ./package.tgz --directory ./_release && rm ./package.tgz",
-    "build": "rimraf ./dist && tsup src/index.ts --format cjs,esm --dts --external @phantom/constants --external @phantom/parsers",
+    "build": "rimraf ./dist && tsc --project . --outDir dist --declaration --emitDeclarationOnly",
```

Comment:
we don't need to use tsup there right? What's the difference?

Reply:
so, this package only exported types, and tsup was creating empty bundle files. And we didn't need it here, as you mention, with tsc is enough